torntools/scripts/content/gym/ttGym.js

```diff
@@ -270,24 +271,21 @@ function displayGraph() {
 						let gains = [];
 						let update_message = `You have gained `;
 
-						if (result.deltaStrength !== 0) {
+						if (result.deltaStrength && result.deltaStrength !== 0) {
 							gains.push(`${numberWithCommas(result.deltaStrength, false)} Strength`);
 						}
-						if (result.deltaDefense !== 0) {
+						if (result.deltaDefense && result.deltaDefense !== 0) {
 							gains.push(`${numberWithCommas(result.deltaDefense, false)} Defense`);
 						}
-						if (result.deltaDexterity !== 0) {
+						if (result.deltaDexterity && result.deltaDexterity !== 0) {
 							gains.push(`${numberWithCommas(result.deltaDexterity, false)} Dexterity`);
 						}
-						if (result.deltaSpeed !== 0) {
+						if (result.deltaSpeed && result.deltaSpeed !== 0) {
```

Comment:
When `result.deltaSpeed` is 0, the `result.deltaSpeed` will already catch it (0 is falsely).

Reply:
Changed.